packages/web-runtime/src/components/Topbar/TopBar.vue

```diff
@@ -5,7 +5,10 @@
     :aria-label="$gettext('Top bar')"
   >
     <div class="oc-topbar-left oc-flex oc-flex-middle oc-flex-between">
-      <applications-menu v-if="appMenuItems.length" :applications-list="appMenuItems" />
+      <applications-menu
+        v-if="appMenuItems.length && mode !== 'embed'"
```

Comment:
Please add a composable that provides a `isEmbeddedMode` (or something like that) - I really don't want this magical untyped string comparison all over the code base

Reply:
Done, added `useEmbedMode` composable into pkg package.